doc/doxygen/install/install-linux.doxygen

```diff
@@ -203,6 +206,14 @@
   mkdir OpenMS-build
   cd OpenMS-build
   \endcode
+  
+  Install the qt4-webkit, qt4-dev-tools and libqtwebkit-dev
+
+   \code
+   sudo apt-get install qt4-webkit
+   sudo apt-get qt4-deb-tools
+   sudo apt-get libqtwebkit-dev
+   \endcode
```

Comment:
I dont think this is the right place for this, there is a description of all required packages further up. However, maybe the names of these packages has changed with different versions of debian/ ubuntu? 

see above where we have the line
```
 sudo apt-get install qt4-default libqtwebkit-dev
```